examples/get_raster_data.sh

```diff
@@ -0,0 +1,7 @@
+mkdir -p data
+cd data
+wget http://s3.amazonaws.com/bokeh_data/geotiff_example.zip
+unzip geotiff_example.zip
+rm geotiff_example.zip
+cd ..
+
```

Comment:
Is there good reason for adding the dataset here instead of in the other python script (noting that this won't work on windows)? No strong feelings here.